gax-grpc/src/test/java/com/google/api/gax/grpc/GrpcDirectStreamingCallableTest.java

```diff
@@ -151,11 +149,10 @@ public void testBidiStreamingClientError() throws Exception {
     latch.await(20, TimeUnit.SECONDS);
     Truth.assertThat(moneyObserver.error).isNotNull();
     Truth.assertThat(moneyObserver.error).isInstanceOf(ApiException.class);
-    Truth.assertThat(((ApiException) moneyObserver.error).getStatusCode().getCode())
-        .isEqualTo(Code.CANCELLED);
-    Truth.assertThat(moneyObserver.response).isNull();
-    StatusException serverReceivedError = (StatusException) serviceImpl.getLastRecievedError();
-    Truth.assertThat(serverReceivedError.getStatus()).isEqualTo(Status.CANCELLED);
+
+    // From gRPC 1.8, when we send an error, the gRPC layer itself wraps our error
+    // and report to server application as "CANCELLED: user cancel before half-close".
```

Comment:
s/report/reports

Reply:
Shouldn't the cause match?

Reply:
@igorbernstein2 I think the problem is more that there's no guarantee what the "proper" behavior is. I think we shouldn't put ourselves in a place where a test can fail every time we update gRPC; does that make any sense? I'll be happy to investigate this more if you think it's important.

Reply:
Could you do what @andreamlin did instead?

https://github.com/andreamlin/gax-java/commit/da281d7b3d81dd769ddb83f9b283d9e02d14b5f7#diff-add2e44a6bc1a86ca4280565b1e34930

(She is removing that diff from her PR so that it is only scoped to Netty removal)

Reply:
Makes sense